api/bases/dataplane.openstack.org_openstackdataplaneservices.yaml

```diff
@@ -30,6 +30,13 @@ spec:
             type: object
           spec:
             properties:
+              ansibleAnyErrorsFatal:
+                type: boolean
+              ansibleMaxFailPercentage:
+                format: int64
```

Comment:
can we just use `int`? I mean, i don't think there's a particular reason to use `int64` (and should be the default anyway on 64bit arch IIRC). If i add to pick a "custom" integer type I'd go with `uint`. WDYT?

Reply:
I changed it to int, I think I just tab completed the suggestion from my editor tbh. I didn't give it too much thought, but int sounds better to me.